src/token/src/canister/inspect.rs

```diff
@@ -115,18 +115,6 @@ fn inspect_message() {
                 ic_cdk::println!("Caller is not allowed to transfer tokens for the requested principal. Rejecting.");
             }
         }
-        "notify" => {
```

Comment:
I think we still want to have this..as we will allow notify..

but now state.ledger.notifications